filter_atts_for_fot.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import numpy as np
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+
+from Ska.Shell import bash
+from Chandra.Time import DateTime
+
+
+def get_options():
+    parser = argparse.ArgumentParser(
+        description="""Make a filtered version of dark_attitudes.dat using the zodiacal brightness
+                       data from the attitude.pl CGI""")
+    parser.add_argument("--date",
+                        required=True,
+                        help="Requested date of expected dark current calibration")
+    opt = parser.parse_args()
+    return opt
+
+
+def filter_atts(date):
+    """
+    Run the attitude.pl Perl CGI with a date and return a filtered version to remake
+    dark_attitudes.dat (since the FOT is already parsing that file, I want to return a
+    file that is formatted the same way).
+
+    :param date: Chandra.Time compatible date
+    :returns: astropy table of dark attitudes filtered by zodiacal brightness limits
+    """
+    date = DateTime(date).date
+
+    Z_LOW = 0
+    Z_UP = 250
+    zodi_annotated_html = bash(
+        'perl ./attitude.pl date="{date}" type="ATT List" z_low="{z_low}" z_up="{z_up}"'.format(
+            date=date[0:8], z_low=Z_LOW, z_up=Z_UP))
+    zodi_list = Table.read(zodi_annotated_html, format='ascii.html')
+    zodi_ok = (zodi_list['Zodi'] >= Z_LOW) & (zodi_list['Zodi'] <= Z_UP)
```

Comment:
IIRC, the Perl script just calculates the zodiacal brightness, and the default "list" output then colorizes the lines with brightness outside the range.  To clean this up, I could either fix the Perl or take a few minutes to write the Python to  read from the zodiacal brightness lookup table directly.